Apps/WebClient/src/ClientApp/src/components/timeline/calendarTimeline.vue

```diff
@@ -68,10 +68,16 @@ export default class CalendarTimelineComponent extends Vue {
                         class="text-center pt-2 noTimelineEntriesText"
                         data-testid="noTimelineEntriesText"
                     >
-                        No Timeline Entries
+                        No records found whitin the selected filters.
```

Comment:
"whitin" --> "within"
Also the "No records found within the selected filters" doesn't sound naturally to me. How about "No records found for your search" or "No records found with the selected filters" ?
@ninosamson @mk08sh  any suggestions?

Reply:
Yes, sorry - "within" was my fault. Should be 
"No records found with the selected filters"